scripts/run-coveralls.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Check code coverage via coveralls
+
+AMICI_PATH="`dirname \"$0\"`"
+AMICI_PATH="`( cd \"$AMICI_PATH/..\" && pwd )`"
+
+# read environment variables put there by build script
+if [ -f ${AMICI_PATH}/scripts/env.sh ]; then
+    . ${AMICI_PATH}/scripts/env.sh
+fi
+
+lcov --base-directory ${AMICI_PATH} --directory ${AMICI_PATH} --zerocounters -q
+
+for MODELSTR in $TESTMODELS; do
+    MODEL=${MODELSTR#model_}
+    cd ${AMICI_PATH}/tests/cpputest/build/${MODEL}
+    ./model_${MODEL}_test
+done
+
+cd ${AMICI_PATH}/build/CMakeFiles/amici.dir/src
```

Comment:
Why do we need to run it in this directory? 

Is there a (simple) way to get nicer file names in the result table (i.e. relative to $AMICI_PATH)?

Reply:
I am currently trying to figure this out, in the lcov report it is fine, but somehow coveralls messes things up, not quite sure why. Overall coveralls is quite poorly documented :(